web/profiles/custom/os2loop/themes/os2loop_theme/os2loop_theme.theme

```diff
@@ -287,3 +289,16 @@ function os2loop_theme_preprocess_form_element(&$variables) {
     $variables['label_display'] = 'after';
   }
 }
+
+/**
+ * Implements hook_preprocess().
+ */
+function os2loop_theme_preprocess_entity_print(&$variables) {
+  $routeName = \Drupal::routeMatch()->getRouteName();
+  if ('entity_print.view.debug' === $routeName) {
+    $variables['node'] = $variables['content'][0][0]['#node'];
```

Comment:
Check that `$variables['content'][0][0]['#node']` is actually set here.

Reply:
Fixed